www/docusaurus.config.js

```diff
@@ -94,6 +94,14 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      announcementBar: {
+        id: 'support_us',
+        content:
+          'Firezone 1.0 is coming! <a target="_blank" rel="noopener noreferrer" href="/beta/signup">Sign up for early access.</a>',
+        backgroundColor: '#fafbfc',
+        textColor: '#091E42',
+        isCloseable: false,
+      },
```

Comment:
Comment out until the form is ready
```suggestion
      /*announcementBar: {
        id: 'support_us',
        content:
          'Firezone 1.0 is coming! <a target="_blank" rel="noopener noreferrer" href="/beta/signup">Sign up for early access.</a>',
        backgroundColor: '#fafbfc',
        textColor: '#091E42',
        isCloseable: false,
      },*/
```